lib/datadog/tracing/contrib/redis/trace_middleware.rb

```diff
@@ -71,10 +71,17 @@ def get_pipeline_commands(commands, command_args)
 
           private
 
-          def resolve(redis_config)
-            custom = redis_config.custom[:datadog] || {}
-
-            Datadog.configuration.tracing[:redis, redis_config.server_url].to_h.merge(custom)
+          def resolve(config)
+            # support both:
```

Comment:
Hey @TonyCTHsu 👋 

Based on your reply, I understand that when using Redis 5+, the following no longer works (and is no longer supported):
```ruby
redis = Redis.new
Datadog.configure_onto(redis, service_name: 'my-redis')
```

I'm updating now the PR based on that :+1:

Reply:
By the way, would we event want to support the following? (just wondering - I prefer to keep that out of the scope of this PR)
```ruby
require 'redis_client'

redis_client = RedisClient.new
Datadog.configure_onto(redis_client, ...)
redis_client.call("PING")
```

My feeling is that, since `configure_onto` is not supported on _redis_ 5+, it would be confusing to have it working with _redis-client_. Also because I believe redis-client can be initialized with custom options... i.e.: 
```ruby
RedisClient.new(custom_options: { datadog: { ... } })
```